### PR TITLE
Added clarifying information to README

### PR DESCRIPTION
**Pull request by metasim**
Stuff that as a new user would have been helpful to have.

Signed-off-by: Simeon H.K. fitch <fitch@astraea.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +96,37 @@ For snapshot integrations use:
   </repository>
 ```
 
+and then include the desired `geomesa-*` dependencies:
+
+```xml
+  <dependency>
+    <groupId>org.locationtech.geomesa</groupId>
+    <artifactId>geomesa-spark</artifactId>
```
Comment (jnh5y) [inline README.md]: Should this be an actual example artifactId?

File: README.md [hunk 2]
```diff
@@ -57,8 +55,7 @@ directory of GeoServer.
 
 ## Maven Integration
 
-GeoMesa artifacts are published to the LocationTech Maven repository. To include GeoMesa in your project,
-add the following repositories to your pom:
+GeoMesa artifacts are published to the [LocationTech Maven repository](https://repo.locationtech.org/content/groups/releases/org/locationtech/geomesa/). To include GeoMesa in your project, add the following repositories to your pom:
```
Comment (jnh5y) [inline README.md]: @metasim These changes should be made to build/README.md as well.  That file is filtered during the build process to update version numbers, etc.

Comment (mdzimmerman) [inline README.md]: It would also be useful if this was put in ``docs/developer`` as well.

File: README.md (hunk 1 above)
Comment (metasim) [inline README.md]: Probably... is that not one? Can you provide an example?

File: README.md [hunk 3]
```diff
@@ -99,6 +96,37 @@ For snapshot integrations use:
   </repository>
 ```
 
+and then include the desired `geomesa-*` dependencies:
```
Comment (jnh5y) [inline README.md]: @metasim I think this block needs to go into the build/README.md; otherwise, it'd get clobbered later...

**Comment by elahrvivaz** [comment]
looks good i'm going to merge. thanks @metasim!

**Comment by elahrvivaz** [comment]
833a4d59c0791090cf8a4abddee359fc59bc491f

**Pull request closed without merging**